### PR TITLE
Improve PFSA logging

### PR DESCRIPTION
**Pull request by gerardgine**
Add volume name and PID to all messages
Remove redundant volume name from some messages

### CODE REVIEW

**Comment by gerardgine** [comment]
I'm not so sure about the change at https://github.com/swiftstack/ProxyFS/pull/410/files#diff-6240498d7b369ef3ab19bbe4dd922dedR26

As @edmc-ss mentioned, we're shipping this conf file as a template in the pfsagent package, and that log path is not consistent with the log paths we use in the controller.

Just defining `LogFilePath` as `pfsagentd.log` will make the log file be at `/pfsagentd.log` in Runway, which is not good either...

File: pfsagentd/log.go [hunk 1]
```diff
@@ -42,8 +44,8 @@ func logf(level string, format string, args ...interface{}) {
 		logMsg         string
 	)
 
-	enhancedFormat = "%-32s %-5s " + format
-	enhancedArgs = append([]interface{}{time.Now().Format(time.RFC3339Nano), level}, args...)
+	enhancedFormat = "[%s][%s][%d][%s] " + format
+	enhancedArgs = append([]interface{}{time.Now().Format(time.RFC3339Nano), globals.config.FUSEVolumeName, unix.Getpid(), level}, args...)
```
Comment (edmc-ss) [inline pfsagentd/log.go]: I'd prefer you call os.Getpid() rather than unix.Getpid(): https://golang.org/pkg/os/#Getpid

**Review by edmc-ss — CHANGES_REQUESTED** [review]
After that change to use os.Getpid(), I'm good with this - thanks!

File: pfsagentd/log.go (hunk 1 above)
Comment (gerardgine) [reply pfsagentd/log.go]: Done! Any thoughts on the change to the config file?

**Pull request merged**